### PR TITLE
Remove no-longer-used Google Drive support from NGINX

### LINKED ISSUE #595: Delete Google Drive support from NGINX config
Now that we use Python to proxy Google Drive files we no longer need the special Google Drive support in NGINX.

If we remove this then the switches associated with `GOOGLE_DRIVE_IN_NGINX` should be removed, as there is only one path now.

## PRs

 * https://github.com/hypothesis/via/pull/608
 * https://github.com/hypothesis/devdata/pull/33

### PR DESCRIPTION
**Pull request by seanh**
Fixes https://github.com/hypothesis/via/issues/595. No rush to merge this, we can wait a few more days until we're more sure of the Python endpoint, but it can be reviewed now.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -28,4 +28,4 @@ COPY . .
 
 USER hypothesis
 
-CMD /usr/bin/envsubst '$${GOOGLE_API_KEY}:$${NGINX_SECURE_LINK_SECRET}' < /var/lib/hypothesis/nginx_envsubst.conf.template > /var/lib/hypothesis/nginx_envsubst.conf && /usr/bin/supervisord -c /var/lib/hypothesis/conf/supervisord.conf
```
Comment (jon-betts) [inline Dockerfile:31]: This is set via `.devdata.env` so I think we can remove that from `bin/devdata.py` and from the devdata repo.

File: conf/nginx/includes/proxy.conf [hunk 2]
```diff
@@ -17,46 +17,10 @@ proxy_ssl_server_name on;
...
-#
-#   The `break` also causes nginx to stop processing at the end of this
-#   `location { ... }` and send the `proxy_pass`'d response back to the
-#   browser, rather than re-starting its processing using the rewritten $uri to
-#   search for a matching `location { ... }` again.
-#
-# http://nginx.org/en/docs/http/ngx_http_rewrite_module.html#rewrite
-# http://nginx.org/en/docs/http/ngx_http_rewrite_module.html#break
-rewrite ^(https)://drive.google.com/uc\?id=(.*)&export=download$ $1://www.googleapis.com/drive/v3/files/$2?key=$google_api_key&alt=media? break;
-
```
Comment (jon-betts) [inline conf/nginx/includes/proxy.conf:52]: Good bye and good riddance.

File: tests/unit/via/services/__init___test.py [hunk 3]
```diff
@@ -43,14 +43,15 @@ def test_it_with_credentials(
...
+        api = create_google_api(sentinel.settings)
 
         load_injected_json.assert_has_calls(
             [
-                call(settings, "google_drive_credentials.json"),
-                call(settings, "google_drive_resource_keys.json", required=False),
+                call(sentinel.settings, "google_drive_credentials.json"),
+                call(
+                    sentinel.settings, "google_drive_resource_keys.json", required=False
+                ),
```
Comment (jon-betts) [inline tests/unit/via/services/__init___test.py]: :+1:

File: Dockerfile (hunk 1 above)
Comment (jon-betts) [reply Dockerfile:31]: I've fixified this

**Review by jon-betts — APPROVED** [review]
I've rebased this and fixed it up. So it's a :+1: from me, but you might want to review the extra bits.

File: Dockerfile (hunk 1 above)
Comment (jon-betts) [reply Dockerfile:31]: PR for devdata here: https://github.com/hypothesis/devdata/pull/33

**Review by marcospri — APPROVED** [review]

**Comment by seanh** [comment]
Rebased

**Pull request merged**